crates/release-automation/src/crate_.rs

```diff
@@ -1,12 +1,16 @@
-use std::collections::{HashMap, HashSet};
-
-use anyhow::bail;
+use anyhow::{bail, Context};
```

Comment:
are the changes in this file a weird merge conflict?

Reply:
Must be because this PR isn't based on develop